java/java.editor.base/test/unit/data/goldenfiles/org/netbeans/modules/java/editor/base/semantic/DetectorTest/testUsedImport129988.pass

```diff
@@ -2,4 +2,5 @@
 [PUBLIC, CLASS, DECLARATION], 4:13-4:29
 [PACKAGE_PRIVATE, METHOD, DECLARATION], 5:9-5:11
 [PUBLIC, CONSTRUCTOR], 6:12-6:28
+[PUBLIC, INTERFACE], 6:29-6:33
```

Comment:
The highlighter originally didn't highlight "List" in "new UsedImport129988<List<String>>();", but that feels like a bug.